doc/faq.md

```diff
@@ -66,16 +67,15 @@ To address the issue director crash or getting upgraded, some implementation int
 2. connection-table is per-lcore, when sync-ed to another DPVS, how to handle per-lcore table in another machine?
 3. "Session-Sharing" works well for "long-connection" but not "short-connection", since the connection's creation/destruction are too frequently to be sync-ed to other directors.
 
-As I know, some L4 LB implemented "session sharing" or "session synchronization", they are configuring same LIPs for each LB director. And each LIP is confgured for one CPU core. Both cases are quite different for DPVS implementation and deployment.
+As I know, some L4 LB implemented "session sharing" or "session synchronization", they are configuring same LIPs for each LB director. And each LIP is configured for one CPU core. Both cases are quite different from DPVS implementation and deployment.
 
 On the other hand, for the high availability of Real Servers, DPVS leverage `keepalived` for health check on RS, both TCP/UDP services can be checked, you can also write your own checking scripts. For more info about health-check, pls refer `LVS`'s document.
 
 <a id="toa" />
 
-### Is DPVS support `TOA`, which `TOA` version is used?
+### Does  DPVS support `TOA`, which `TOA` version is used?
 
-Yes, and we use same `toa` version as [alibaba/LVS](https://github.com/alibaba/LVS). It's open-sourced, to fetch Real Server `toa.ko` module, pls check [#issue-70](https://github.com/iqiyi/dpvs/issues/70).
-DPVS side TOA format is defined in `include/ipvs/proto_tcp.h`, while option code is `254`, option length is 8.
+Yes, and DPVS's `toa` derives from the open-sourced [alibaba/LVS](https://github.com/alibaba/LVS). The RS's toa kernel module implementation is at [kmod/toa](../kmod/toa). Compare to the original toa, DPVS toa add supports for IPv6 and Nat64. DPVS side TOA format is defined in [proto_tcp.h](../include/ipvs/proto_tcp.h), while option code is `254`, option length is 8.
```

Comment:
**Compared** to the original toa, **DPVS's** toa add supports for IPv6 and Nat64. TOA format **on the side of DPVS** is defined in [proto_tcp.h](../include/ipvs/proto_tcp.h), while **the** option code is `254` **and the** option length is 8.